index.html

```diff
@@ -5,10 +5,21 @@
     <title>All-Inclusive-Web-Team</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <nav>
-      <img id='logo' src="assets/dcrdlogo.png" alt="logo">
-    </nav>
+    <div id="nav-bar">
+      <ul>
+        <li id="studentEl">Students</li>
+        <li id="teacherEl">Teachers</li>
+        <img id="logo" src="./assets/dcrdlogo.png">
```

Comment:
This `<img>` tag must be wrapped in an `<li>` because it's in a `<ul>` ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#try_it) see "Permitted content", [WHATWG](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element) see "Content model").